test/xspec.bats

```diff
@@ -326,3 +326,20 @@ teardown() {
 	grep '<style type="text/css">' ../tutorial/xspec/escape-for-regex-result.html
 	grep 'margin-right:' ../tutorial/xspec/escape-for-regex-result.html
 }
+
+
+@test "running XSpec via ant for XSLT support" {
+    run ant -buildfile ${PWD}/../build.xml -Dxspec.xml=${PWD}/../tutorial/escape-for-regex.xspec -lib /tmp/xspec/saxon/saxon9he.jar -lib /tmp/ant/lib/xmltask.jar -lib /tmp/ant/lib/ant-contrib.jar 
```

Comment:
* `-lib /tmp/xspec/saxon/saxon9he.jar` should be `-lib ${SAXON_CP}` to coordinate with other tests and help changing Saxon versions/editions.
* `-lib /tmp/ant/lib/xmltask.jar -lib /tmp/ant/lib/ant-contrib.jar` may be shortened to `-lib /tmp/ant/lib`
* `-Dclean.output.dir=true` should be added, otherwise `git status` finds a temp file after testing locally.

Reply:
I implemented this feedback in https://github.com/xspec/xspec/commit/cd327716028c1071ec2f4a89c4cc072fb98e65a0